### DISCUSSION TITLE
CefSharp.BrowserSubporcess.exe Version 45.0.0.0

### DISCUSSION THREAD

**Discussion by irjadah-dev** [cefsharp/CefSharp#3450] [General]
.NET 4.0
WPF
x86 
VB.NET Project

Hi,
Im using the Browser to play Webm Video on a loop. After 2-3 Hours the Browser freezes and the Suprocess.exe crashes. in the Logfile from the Application cant seee anything. 
Thank You!

Here is the Eventlog :
_Name der fehlerhaften Anwendung: CefSharp.BrowserSubprocess.exe, Version: 45.0.0.0, Zeitstempel: 0x5652f7fb
Name des fehlerhaften Moduls: libcef.dll, Version: 3.2454.1344.0, Zeitstempel: 0x562d8f27
Ausnahmecode: 0x4000001f
Fehleroffset: 0x00186429
ID des fehlerhaften Prozesses: 0x1e58
Startzeit der fehlerhaften Anwendung: 0x01d71594c315d56c
Pfad der fehlerhaften Anwendung: C:\Program Files (x86)\TTWin\CefSharp.BrowserSubprocess.exe
Pfad des fehlerhaften Moduls: C:\Program Files (x86)\TTWin\libcef.dll
Berichtskennung: d3cf7ede-89a0-4b67-8e9c-23d0565518d1
Vollständiger Name des fehlerhaften Pakets: 
Anwendungs-ID, die relativ zum fehlerhaften Paket ist:_

**Comment by amaitland** [comment]
Version 45 is old and unsupported. In future please only option an issue if you can reproduce the problem with the current version. https://github.com/cefsharp/CefSharp/releases

My suggests would be:
- Make sure your application is large address aware, see https://github.com/cefsharp/CefSharp/wiki/General-Usage#win32-out-of-memory
- Use DevTools to identify and fix any memory leaks in your JavaScript. 

If the problem is a memory leak in Chromium you'll need to upgrade to a newer version.
